restygwt/src/test/java/org/fusesource/restygwt/client/codec/PolymorphicEncoderDecoderTestGwt.java

```diff
@@ -100,6 +100,8 @@ public void testJsonSubTypesWithInterfaceUsingConcreteImplementationCodec() {
         JSONValue json = codec.encode(o1);
         JSONValue objectClass = json.isObject().get("@class");
         assertNotNull(objectClass);
+        assertEquals(DefaultImplementationOfSubTypeInterface.class.getName().replace("$","."),
```

Comment:
PR #272 relaxed this unit test by removing an asserting, it can be reintroduced now and still passes